util/statvar_dcid_generator.py

```diff
@@ -0,0 +1,139 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import copy
+import re
+
+
+class StatVarDcidGenerator:
```

Comment:
Given there are no constructor args, not sure why this should be a class. Can this just be a: generate_stat_var_dcid() function?

Reply:
The latest commit added some constructor args. The ```IGNORE_PROPS``` and ```STAT_TYPE_LOOKUP``` are initialised in the constructor. 

I made it into a class so that it can be inherited and the methods can be changed depending on the dataset/table. Let me know if just making it a function would be better.